services/QuillLMS/engines/evidence/lib/evidence/anthropic/chat.rb

```diff
@@ -80,9 +82,23 @@ def request_body
       def self.chat_api? = true
 
       private def extract_text_response
-        response
-          .parsed_response['content']
-          .first['text']
+        parsed = response.parsed_response
```

Comment:
Unless you need these very specific errors, you might simplify with a generic catchall, e.g.

```ruby
response
  .parsed_response['content']
   .first['text']
rescue TypeError, NoMethodError => e
  raise CleanedResultsError, "Invalid response format: #{e.message}"